jekyll/_cci2/schedule-pipelines-with-multiple-workflows.adoc

```diff
@@ -17,22 +17,34 @@ contentTags:
 [#introduction]
 == Introduction
 
-As a scheduled pipeline is a triggered pipeline, and it will run every workflow in the configuration. You can set conditionals to keep the scheduled pipelines running only when you want them to run. One way is through built-in pipeline values, and one way is through custom pipeline parameters.
+As a scheduled pipeline is a triggered pipeline, and it will run every workflow in the configuration. You can filter workflows with set conditionals to keep the scheduled pipelines running only when you want them to run. One way is through built-in pipeline values, and one way is through custom pipeline parameters.
 
-[#filter-using-built-in-pipeline-values]
-## Filter using built-in pipeline values
+[#schedule-using-built-in-pipeline-values]
+## Schedule using built-in pipeline values
 
-One way to implement workflows filtering is by using pipeline values. The example below uses the built in pipeline values `pipeline.trigger_source` and `pipeline.schedule.name`.
+One way to implement workflows filtering is by using pipeline values. The example below uses the built in pipeline values `pipeline.trigger_source` and `pipeline.schedule.name`. The examples use `daily_build` and `nightly_build` as the `pipeline.schedule.name`, however, the value of `pipeline.schedule.name` can be whatever you would like when using scheduled pipelines.
```

Comment:
Can we add explanations too like "In this example the workflow `daily-run-workflow` will only run then the triggered scheduled pipeline is called `daily_build`" ... if I'm understanding correctly?